semeio/workflows/misfit_preprocessor/config.py

```diff
@@ -251,51 +252,40 @@ def _inject_default_t(config):
             cs.MetaKeys.ElementValidators: (_one_of(SPEARMAN_CORRELATION, AUTO_SCALE),),
             cs.MetaKeys.Default: AUTO_SCALE,
         },
-        SPEARMAN_CORRELATION: SPEARMAN_CORRELATION_SCHEMA,
-        AUTO_SCALE: AUTO_SCALE_SCHEMA,
+        "clustering": HIERARCHICAL_SCHEMA,
+        "pca": PCA_SCHEMA,
     },
 }
 
 
-_SCALING_SCHEMA = {
+@cs.transformation_msg("Detecting method.")
+def insert_method(layer):
+    if "method" in layer:
+        if len(layer) == 1:
+            return layer
+        if len(layer) == 2 and layer["method"] in layer:
+            return layer
+        raise ValueError(f"Found method in {layer}. Did not match {layer.keys}.")
```

Comment:
Perhaps improve this error message? Is a bit cryptic.

Reply:
We just made it also!